packages/Console/src/Application/Plugin/ContextualHelpPlugin.php

```diff
@@ -49,7 +49,7 @@ function (BeforeActionEvent $e) {
             ->on(
                 Executor::EVENT_INVALID_USAGE,
                 function (InvalidUsageEvent $e) {
-                    if ($e->getInputSequence()->count() > 1) {
+                    if ($e->getInputSequence()->count() > 2 || is_null($e->getOptions()->find('help'))) {
```

Comment:
This doesn't work .... misunderstanding of what OptionDefinitionCollection represented. Help is always present in the _option definitions_.